index.html

```diff
@@ -28,11 +28,19 @@ <h1><a href="#0">BCIT</a></h1>
    <div class="wrapper">
        <main>
            <div class="top-banner">
-               <h1>COMP1950</h1>
-           </div>
+               <!--<h1>COMP1950</h1>-->
+           
+
+           <div class="banner-wrapper">
+                <p class="title">Welcome to</p>
+                <div class="clip-text clip-text_one clip-text--cover">COMP1950</div>
+            </div>
+
            <div class="mid-banner"></div>
```

Comment:
I don't see these 2 classes being used yet. Are these supposed to be here?

Reply:
Yes, It supposed to be here because I reserved them for additional content on home page.